images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -98,16 +98,29 @@ RUN ${IMAGES_DIR}/retry.sh "curl -o /usr/local/share/ca-certificates/verisign.cr
     
 # Install PHP pre-reqs	# Install PHP pre-reqs
 RUN if [ "${DEBIAN_FLAVOR}" = "buster" ]; then \
-    apt-get update \
-    && apt-get upgrade -y \
-    && apt-get install -y \
-        $PHPIZE_DEPS \
-        ca-certificates \
-        curl \
-        xz-utils \
-        libsodium-dev \
-        libncurses5 \
-    --no-install-recommends && rm -r /var/lib/apt/lists/* ; \
+        apt-get update \
+        && apt-get upgrade -y \
+        && apt-get install -y \
+            $PHPIZE_DEPS \
+            ca-certificates \
+            curl \
+            xz-utils \
+            libsodium-dev \
+            libncurses5 \
+        --no-install-recommends && rm -r /var/lib/apt/lists/* \
+        # A workaround for php 8.0 composer issue: https://github.com/microsoft/Oryx/issues/1100
+        && ln -s /usr/lib/x86_64-linux-gnu/libonig.so /usr/lib/x86_64-linux-gnu/libonig.so.4 \
+        && apt-get update -y \
+        && apt-get install software-properties-common -y \
+        && apt-get update -y \
+        && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D1DAC98AF575D16E \
```

Comment:
Do you know if this Ubuntu key is something that might be only temporary? Also, is this a key generated specifically for us or is it commonly available?

Reply:
The ubuntu key is used for the source http://ppa.launchpad.net/xapienz/curl34/ubuntu, which is similarly as signing, it's commonly available and not a temporary key.

Reply:
Thanks for validating